src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -734,6 +733,73 @@ public void Purge(Document document)
             });
         }
 
+        /// <summary>
+        /// Purges the given document id of the <see cref="Document"/> 
+        /// from the database.  This leaves no trace behind and will 
+        /// not be replicated
+        /// </summary>
+        /// <param name="docId">The id of the document to purge</param>
+        /// <exception cref="C4ErrorCode.NotFound">Throws NOT FOUND error if the document 
+        /// of the docId doesn't exist.</exception>
+        [ContractAnnotation("null => halt")]
+        public void Purge(string docId)
+        {
+            CBDebug.MustNotBeNull(Log.To.Database, Tag, nameof(docId), docId);
+            var document = GetDocument(docId);
+            Purge(document);
+        }
+
+        /// <summary>
+        /// Sets an expiration date on a document. After this time, the document
+        /// will be purged from the database.
+        /// </summary>
+        /// <param name="docId"> The ID of the <see cref="Document"/> </param> 
+        /// <param name="timestamp"> Nullable expiration timestamp as a 
+        /// <see cref="DateTimeOffset"/>, set timestamp to <c>null</c> 
+        /// to remove expiration date time from doc.</param>
+        /// <returns>Whether succesfully sets an expiration date on the document</returns>
+        /// <exception cref="C4ErrorCode.NotFound">Throws NOT FOUND error if the document 
```

Comment:
The cref here should be an exception (CouchbaseLiteException) since that is what gets thrown.